External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -184,7 +184,8 @@ static public bool OnChar(ScintillaControl Sci, int Value, bool autoHide)
                         if (features.hasGenerics && position > 2)
                         {
                             char c0 = (char)Sci.CharAt(position - 2);
-                            if (c0 == '.' /*|| Char.IsLetterOrDigit(c0)*/)
+                            //TODO: We should check if we are actually on a generic type
```

Comment:
The change was fine in all the tests I made (plus the build used by my workmates has this change, and nothing strange was detected in all this time), but I don't like this kind of simplistic check, there are cases where c0 may not return true and we are still in a generic, also, on current development branch if you try in Haxe to write .<, no matter where you are, it will always display the completion list with the available classes.that's the reason for the comment. This is the reason of the comments, the change is an improvement, but not a proper fix.